static/js/stream_settings_ui.js

```diff
@@ -153,6 +153,7 @@ export function toggle_pin_to_top_stream(sub) {
 }
 
 let subscribed_only = true;
+let unsubscribed_only = true;
```

Comment:
Do you think we should refactor these to `show_subscribed` and `show_unsubscribed`?  Probably makes sense to do as a follow-up refactor, but this model feels feel a bit weird ("All streams" has both of these false).

Reply:
Yea, I think we should refactor the names since `subscribed_only` and `unsubscribed_only` sounds a bit confusing to me. Do you suggest any different model other than having these two variables?